src/golang/cmd/server/handler/v2/dag_get.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 // This file should map directly to
-// src/ui/common/src/handlers/DagGet.tsx
+// src/ui/common/src/handlers/v2/DagGet.tsx
```

Comment:
Curious, when you run godoc, does this actually link to the file?
Or are IDEs able to to follow this and go directly there?

Reply:
nope there's no automated navigation :(